app/utils/tests/injectSaga.test.js

```diff
@@ -1,131 +1,137 @@
-/* eslint-disable import/namespace */
 /* eslint-disable no-import-assign */
+/* eslint-disable react/display-name */
```

Comment:
Why are we disabling eslint rules ?

Reply:
import/no-name-as-default doesn’t work with eslint 9
https://github.com/import-js/eslint-plugin-import/issues/2995

So for now we need to comment it, till the community resolves it